### DISCUSSION TITLE
Any way to have a backup WM

### DISCUSSION THREAD

**Discussion by PerfMonk** [hyprwm/Hyprland#2497] [General]
I would like to keep Plasma as a backup WM but Hyprland complain that I should delete xdg-desktop-portal-kde.  I can't since it's needed by KDE.  I use to kill xdg-desktop-portal-kde when starting Hyprland and launch xdg-desktop-portal-hyprland.  Is there any way to manage those conflicts ???

**Comment by vaxerski** [comment]
Not exactly. The issue is [this](https://github.com/flatpak/xdg-desktop-portal/issues/969)

If you can have both installed, launch hyprland, and be able to screenshare and stuff you can suppress the warning in the hyprland config and keep both.

However, on certain systems, having both will make xdp die (aka. 30s launches of apps, no screensharing, etc)

**Comment by donovanglover** [comment]
If you're using NixOS, this is easy to achieve with either multiple [nixosConfigurations](https://nixos.wiki/wiki/Flakes#Using_nix_flakes_with_NixOS) or through the usage of [specializations](https://nixos.wiki/wiki/Specialisation). For example, see [plasma.nix](https://github.com/donovanglover/nix-config/commit/798aa35edf440d0cf4a25b63a4fa77e9f1d95371) in my config.

This effectively lets you switch between as many window managers / desktop environments as you want without any of them conflicting with each other.
